### PR TITLE
Created fallback docs if GitHub API request fails.

### PR DESCRIPTION
**Pull request by AlexanderWert**
fixes: #171

### CODE REVIEW

**Review by astorm — APPROVED** [review]
👍 Does the thing it says it does.  When API limits have been reached this is what I see

what-i-see

Approving.

**Review by bmorelli25 — APPROVED** [review]
🥇

File: docs/lambda-selector/lambda-attributes-selector.asciidoc [hunk 1]
```diff
@@ -106,7 +127,9 @@ window.addEventListener("DOMContentLoaded", async () => {
 });
 </script>
 
-<div role="lambda-selector">
+<p id="fallback-extension-arn-selector-section">Pick the right ARN from <a target="_bkank" href="RELEASE_LINK">this release table for the APM Lambda Extension Layer</a>.</p>
+<p id="fallback-agent-arn-selector-section">In addition, pick the right ARN from <a target="_bkank" href="RELEASE_LINK">this release table for the APM Agent Layer</a>.</p>
```
Comment (trentm) [inline docs/lambda-selector/lambda-attributes-selector.asciidoc:131]:
```suggestion
<p id="fallback-extension-arn-selector-section">Pick the right ARN from <a target="_blank" href="RELEASE_LINK">this release table for the APM Lambda Extension Layer</a>.</p>
<p id="fallback-agent-arn-selector-section">In addition, pick the right ARN from <a target="_blank" href="RELEASE_LINK">this release table for the APM Agent Layer</a>.</p>
```

Were these intended to be `target="_blank"`?

**Comment by bmorelli25** [comment]
Merging so I can backport with #158.

**Comment by bmorelli25** [comment]
oh wow

**Comment by bmorelli25** [comment]
that was awful timing on my part

File: docs/lambda-selector/lambda-attributes-selector.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/lambda-selector/lambda-attributes-selector.asciidoc:131]: Fixed on `main` in d0f03e72b9e5d467dd375a60fc8ef98a1d34b29c.

**Pull request merged**